apluslms-yamlidator/apluslms_yamlidator/validator.py

```diff
@@ -10,6 +10,7 @@
     ValidationError,
     validators,
 )
+from jsonschema._utils import types_msg
```

Comment:
Create local implementation, as this is import from a "private" module, which can change at any time.